EX1/HW5.py

```diff
@@ -0,0 +1,118 @@
+import os
+import datetime
+import random
+import re
+
+
+
+def get_files_in_directory(path):
+    file_names = []
+    with os.scandir(path) as directory:
+        for entry in directory:
```

Comment:
You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs.